template/functions/card.php

```diff
@@ -0,0 +1,53 @@
+<?php
```

Comment:
Anything under the `/template` directory should only be globally applicable utilities, i.e. something that we could PR back into the framework repo. This file should be moved under the `/about` directory